### ISSUE TITLE
Update Syntax for `compute_environments`

### ISSUE THREAD

**Issue by schammah** [terraform-aws-modules/terraform-aws-batch#30]
## Is your request related to a new offering from AWS?
Is this functionality available in the [AWS provider for Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)? See [CHANGELOG.md](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md), too.
  - Yes ✅:

## Is your request related to a problem? Please describe.

```
 on .terraform/modules/batch/main.tf line 231, in resource "aws_batch_job_queue" "this":
│  231:   compute_environments  = slice([for env in try(each.value.compute_environments, keys(var.compute_environments)) : aws_batch_compute_environment.this[env].arn], 0, min(length(try(each.value.compute_environments, keys(var.compute_environments))), 3))
│
│ This parameter will be replaced by `compute_environment_order`.
```

## Describe the solution you'd like.
Update the resource to `compute_environment_order`

## Additional context
Current Parameter will be deprecated soon, written also in the [official doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/batch_job_queue)

**Comment by schammah** [comment]
does this makes sense?

**Comment by bryantbiggs** [comment]
it makes sense, but its a breaking change. right now its just a warning that the argument will be removed in the next major version - is there an issue that you are running into?

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.0.0 :tada:
